core/register_command.php

```diff
@@ -186,6 +186,7 @@
 	$application->add(new OC\Core\Command\Group\ListCommand(\OC::$server->getGroupManager()));
 	$application->add(new OC\Core\Command\Group\AddUser(\OC::$server->getUserManager(), \OC::$server->getGroupManager()));
 	$application->add(new OC\Core\Command\Group\RemoveUser(\OC::$server->getUserManager(), \OC::$server->getGroupManager()));
+	$application->add(new OC\Core\Command\Group\Info(\OC::$server->getGroupManager()));
```

Comment:
```suggestion
	$application->add(new OC\Core\Command\Group\Info(\OC::$server->get(\OCP\IGroupManager::class)));
```

the specific getters are deprecated, you can go straigh away with ↑. Even import  the use statement. But since all the files is like this yet, it's fine as well – then we do it at once at some point.